horizontal-profile/FISViewController.m

```diff
@@ -0,0 +1,195 @@
+@property (weak, nonatomic) IBOutlet UITextView *myTextView;
+
+@property (strong, nonatomic) IBOutletCollection(UIImageView) NSArray *myImageViews;
+@property (strong, nonatomic) IBOutletCollection(UILabel) NSArray *myLabels;
+
+@property (strong, nonatomic) NSLayoutConstraint *coverImageHeightPortrait;
+@property (strong, nonatomic) NSLayoutConstraint *coverImageHeightLandscape;
+
+@property (strong, nonatomic) NSArray *textView6PlusLandscape;
+@property (strong, nonatomic) NSArray *textView6PlusPortrait;
+
+@end
+
+@implementation FISViewController
+
+- (void)viewDidLoad
+{
+    [super viewDidLoad];
+    
+    // remove constraints
+    [self removeAllConstraints];
+    
+    // setup images
+    [self.coverImage setImage: [UIImage imageNamed:@"stuffmomnevertoldyou.jpg"]];
+    [self.profileImage setImage: [UIImage imageNamed:@"Blake_Farenthold.jpg"]];
+    
+    // set constraints
+    [self setPortraitConstraints];
+}
+
+- (void)didReceiveMemoryWarning
+{
+    [super didReceiveMemoryWarning];
+}
+
+-(void)viewDidLayoutSubviews
+{
+    [super viewDidLayoutSubviews];
+}
+
+-(void)willTransitionToTraitCollection:(UITraitCollection *)newCollection withTransitionCoordinator:(id<UIViewControllerTransitionCoordinator>)coordinator
+{
+    //         if UITraitNameVerticalSizeClass = Compact >>> LANDSCAPE
+    //                          "              = Regular >>>
+    // nab 6+:   if newCollection.displayScale = 3.000
+    
+    
+    if(newCollection.displayScale == 3.0) // IPHONE 6 PLUS
+    {
+        if (newCollection.verticalSizeClass == UIUserInterfaceSizeClassRegular) // PORTRAIT
+        {
+            [self.view removeConstraints: self.textView6PlusLandscape];
+            [self.view addConstraints: self.textView6PlusPortrait];
+        }
+        else if (newCollection.verticalSizeClass == UIUserInterfaceSizeClassCompact) // LANDSCAPE
+        {
+            [self.view removeConstraints: self.textView6PlusPortrait];
+            [self.view addConstraints: self.textView6PlusLandscape];
+        }
+    }
+    else
+    {
+        if (newCollection.verticalSizeClass == UIUserInterfaceSizeClassRegular) // PORTRAIT
+        {
+            [self.view removeConstraint: self.coverImageHeightLandscape];
+            [self.view addConstraint: self.coverImageHeightPortrait];
+        }
+        else if (newCollection.verticalSizeClass == UIUserInterfaceSizeClassCompact) // LANDSCAPE
+        {
+            [self.view removeConstraint: self.coverImageHeightPortrait];
+            [self.view addConstraint: self.coverImageHeightLandscape];
+        }
+    }
+    NSLog(@"transitioning to: %@", newCollection);
+    
+    [self.view layoutSubviews];
+}
+
+
+-(void) setCoverImageConstraints
+{
+    NSLayoutConstraint *pinToTop = [NSLayoutConstraint constraintWithItem:self.coverImage attribute:NSLayoutAttributeTop relatedBy:NSLayoutRelationEqual toItem:self.view attribute:NSLayoutAttributeTop multiplier:1.0 constant:0];
```

Comment:
no prob. 

wanna share that plug-in that you used to use in class for that? would make my life easier :)